dev_tools/bash_scripts_test.py

```diff
@@ -238,19 +263,24 @@ def test_pytest_and_incremental_coverage_branch_selection():
     assert result.err == "Comparing against revision 'HEAD'.\n"
 
     result = run(script_file='check/pytest-and-incremental-coverage',
+                 tmpdir_factory=tmpdir_factory,
                  arg='HEAD~999999')
     assert result.exit_code == 1
     assert result.out == ''
     assert "No revision 'HEAD~999999'." in result.err
 
-    result = run(script_file='check/pytest-and-incremental-coverage')
+    result = run(
+        script_file='check/pytest-and-incremental-coverage',
+        tmpdir_factory=tmpdir_factory,
```

Comment:
Nit: Remove the trailing comma to make yapf format this the same as all the other `run` calls.

Reply:
Done.